scss/6-components/_contact_details.scss

```diff
@@ -1,4 +1,6 @@
 .cads-contact-details {
+  @extend %prose;
```

Comment:
Rather than extending here would it make sense to add `cads-prose` explicitly to the contact details template? Personally find that clearer for these kinds of scoped styles.